gapic/packaging/api_defaults.yaml

```diff
@@ -27,13 +27,22 @@ generated_package_version:
   #       deleted but files still exist on the pypi servers and can prevent
   #       us from using the version number again.
   python:
-    lower: '0.15.4'
-    upper: '0.16dev'
+    default:
+      lower: '0.15.4'
+      upper: '0.16dev'
```

Comment:
One option to not be disruptive is to allow both old form (lower/upper) and new form (ga:lower/upper)